receiver/nsxtreceiver/README.md

```diff
@@ -0,0 +1,66 @@
+# NSX-T Receiver
+
+This receiver fetches metrics important to run virtual networking using NSX-T. The receiver ingests metrics via the [NSX Rest API](https://docs.vmware.com/en/VMware-NSX-Data-Center-for-vSphere/6.4/nsx_64_api.pdf).
+
+Supported pipeline types: `metrics`
+
+> :construction: This receiver is in **BETA**. Configuration fields and metric data model are subject to change.
```

Comment:
Based on recent defined [component stability guidelines](https://github.com/open-telemetry/opentelemetry-collector-contrib#stability-levels), this component should be considered as Alpha.